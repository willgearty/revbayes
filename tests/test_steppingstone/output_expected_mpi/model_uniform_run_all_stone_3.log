Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21390.51	-21378.08	-12.43353	0.7000234	0.112389	0.0865092	0.183111	0.0439238	0.114046	0.246037	0.438426	0.129027	0.221579	0.103341	0.260316	0.138855	0.136917	0.266264	0.182472	0.174002	0.147719	0.138195	0.169869	0.198678	0.380606	0.194694	0.354403	0.0430078	0.0502588	0.00289414	0.0735366	0.0656154	0.0325751	0.234895	0.1348	0.08307	0.111712	0.0165138	0.0247186	0.0231285	0.035484	0.0479715	0.0710057	0.0137004	0.255873	0.14648	0.338844	0.0225354	0.336783	0.0498243	0.0448881	0.478463	0.0675063	0.0741933	0.363541	0.924877	2.63739	0.378685	0.334896	0.0702697	0.216149	0.3004459	6.227463
10	-21371.79	-21355.93	-15.86198	0.845409	0.146058	0.0631734	0.223321	0.069205	0.200087	0.194602	0.315446	0.133798	0.0759248	0.110292	0.226048	0.159458	0.0974703	0.233833	0.250991	0.121419	0.152428	0.193576	0.148451	0.262512	0.349645	0.212664	0.300792	0.124484	0.0512747	0.00130197	0.153242	0.0863966	0.042887	0.395557	0.157583	0.145363	0.125915	0.025137	0.0700806	0.0567177	0.0252787	0.0317672	0.109935	0.015711	0.118206	0.176513	0.449146	0.0246724	0.30443	0.0596693	0.0381975	0.529615	0.0434162	0.105178	0.424793	0.968784	2.50124	0.383524	0.330213	0.0691861	0.217077	0.3897326	6.603691
