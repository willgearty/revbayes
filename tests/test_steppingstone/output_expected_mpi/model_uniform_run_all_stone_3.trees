Iteration	Posterior	Likelihood	Prior	psi
0	-21390.51	-21378.08	-12.43353	(Daubentonia_madagascariensis[&index=7]:0.438426,Galeopterus_variegatus[&index=23]:0.354403,((((Colobus_guereza[&index=6]:0.246037,(Chlorocebus_aethiops[&index=5]:0.114046,Macaca_mulatta[&index=13]:0.136917)[&index=25]:0.050259)[&index=42]:0.146480,(Pan_paniscus[&index=17]:0.147719,Hylobates_lar[&index=9]:0.221579)[&index=32]:0.083070)[&index=30]:0.234895,(Callicebus_donacophilus[&index=2]:0.086509,((Saimiri_sciureus[&index=20]:0.198678,Cebus_albifrons[&index=3]:0.183111)[&index=26]:0.002894,Aotus_trivirgatus[&index=1]:0.112389)[&index=33]:0.111712)[&index=43]:0.338844)[&index=31]:0.134800,(Tarsius_syrichta[&index=21]:0.380606,(((Loris_tardigradus[&index=12]:0.138855,Nycticebus_coucang[&index=15]:0.182472)[&index=37]:0.035484,((Otolemur_crassicaudatus[&index=16]:0.174002,Galago_senegalensis[&index=8]:0.129027)[&index=24]:0.043008,Perodicticus_potto[&index=18]:0.138195)[&index=29]:0.032575)[&index=41]:0.255873,(((Cheirogaleus_major[&index=4]:0.043924,Lepilemur_hubbardorum[&index=11]:0.260316)[&index=35]:0.024719,Microcebus_murinus[&index=14]:0.266264)[&index=39]:0.071006,((Varecia_variegata_variegata[&index=22]:0.194694,Lemur_catta[&index=10]:0.103341)[&index=38]:0.047972,Propithecus_coquereli[&index=19]:0.169869)[&index=40]:0.013700)[&index=27]:0.073537)[&index=28]:0.065615)[&index=34]:0.016514)[&index=36]:0.023128)[&index=44];
10	-21371.79	-21355.93	-15.86198	(((Tarsius_syrichta[&index=21]:0.349645,((Propithecus_coquereli[&index=19]:0.148451,((Varecia_variegata_variegata[&index=22]:0.212664,Lemur_catta[&index=10]:0.110292)[&index=38]:0.031767,((Cheirogaleus_major[&index=4]:0.069205,Microcebus_murinus[&index=14]:0.233833)[&index=39]:0.109935,Lepilemur_hubbardorum[&index=11]:0.226048)[&index=35]:0.070081)[&index=40]:0.015711)[&index=27]:0.153242,((Nycticebus_coucang[&index=15]:0.250991,Loris_tardigradus[&index=12]:0.159458)[&index=37]:0.025279,(Perodicticus_potto[&index=18]:0.193576,(Otolemur_crassicaudatus[&index=16]:0.121419,Galago_senegalensis[&index=8]:0.133798)[&index=24]:0.124484)[&index=29]:0.042887)[&index=41]:0.118206)[&index=28]:0.086397)[&index=36]:0.056718,Daubentonia_madagascariensis[&index=7]:0.315446)[&index=34]:0.025137,Galeopterus_variegatus[&index=23]:0.300792,((Callicebus_donacophilus[&index=2]:0.063173,(Cebus_albifrons[&index=3]:0.223321,(Saimiri_sciureus[&index=20]:0.262512,Aotus_trivirgatus[&index=1]:0.146058)[&index=26]:0.001302)[&index=33]:0.125915)[&index=43]:0.449146,((Pan_paniscus[&index=17]:0.152428,Hylobates_lar[&index=9]:0.075925)[&index=32]:0.145363,(Colobus_guereza[&index=6]:0.194602,(Macaca_mulatta[&index=13]:0.097470,Chlorocebus_aethiops[&index=5]:0.200087)[&index=25]:0.051275)[&index=42]:0.176513)[&index=30]:0.395557)[&index=31]:0.157583)[&index=44];
