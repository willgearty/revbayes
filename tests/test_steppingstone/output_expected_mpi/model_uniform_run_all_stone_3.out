state	power	likelihood
4	0.16807	-21396.2
5	0.16807	-21392.9
6	0.16807	-21384.2
7	0.16807	-21375.9
8	0.16807	-21380.8
9	0.16807	-21374.9
10	0.16807	-21355.9
